modules/redis/src/main/scala/zio/redis/options/PubSub.scala

```diff
@@ -0,0 +1,20 @@
+package zio.redis.options
+
+import zio.UIO
+import zio.redis.internal.RespValue
+object PubSub {
```

Comment:
Add an empty line between the import block and the object.

Reply:
I added https://github.com/zio/zio-redis/pull/714/commits/1a0862f7b30f2dc8a611217c98e8f85814276f2b